tests/benchmarks/data_structure_perf/rbtree_perf/src/rbtree_perf.c

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright (c) 2017 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <ztest.h>
+#include <sys/rb.h>
+
+#define TREE_SIZE 512
+
+/* rbnode structure is embeddedable in user structure */
+struct container_node {
+	struct rbnode node;
+	int value;
+};
+static struct rbnode nodes[TREE_SIZE];
+static struct rbtree tree;
+
+/* Our "lessthan" is just the location of the struct */
+bool node_lessthan(struct rbnode *a, struct rbnode *b)
+{
+	return a < b;
+}
+
+/**
+ * @brief Test whether rbtree node struct is embedded
+ * in any user struct
+ *
+ * @details Initialize an user defined structure contains
+ * rbtree node.And into a rbtree and enumerate
+ * the rbtree nodes.
+ *
+ * verify that the value enumerated is equal to the
+ * value initialized.If the verification pass,the user
+ * defined structure works.
+ * verify "for each" style APIs work.
+ *
+ * @ingroup lib_rbtree_tests
+ *
+ * @see RB_FOR_EACH(),RB_FOR_EACH_CONTAINER()
+ */
+void test_rbtree_container(void)
+{
+	int count = 0;
+	static struct rbtree test_tree_l;
+	static struct container_node tree_node[10];
+	static struct container_node *c_foreach_node;
+	static struct rbnode *foreach_node;
```

Comment:
Do these variables need to be static ?

Reply:
Double check it and I think it's local variables and do not need to return, so I don’t think need to add 'static'. And I have removed it, thanks.